src/core_plugins/kibana/server/tutorials/apm/apm_server_instructions.js

```diff
@@ -82,20 +98,31 @@ export function createWindowsServerInstructions() {
   return [
     {
       title: createDownloadServerTitle(),
-      textPre:
-        '1. Download the APM Server Windows zip file from the [Download page](https://www.elastic.co/downloads/apm/apm-server).\n' +
-        '2. Extract the contents of the zip file into `C:\\Program Files`.\n' +
-        '3. Rename the `apm-server-{config.kibana.version}-windows` directory to `APM-Server`.\n' +
-        '4. Open a PowerShell prompt as an Administrator (right-click the PowerShell icon and select' +
-        ' **Run As Administrator**). If you are running Windows XP, you might need to download and install PowerShell.\n' +
-        '5. From the PowerShell prompt, run the following commands to install APM Server as a Windows service:',
+      textPre: i18n.translate('kbn.server.tutorials.apm.windowsServerInstructions.textPre', {
+        // eslint-disable-next-line no-multi-str
+        defaultMessage: '1. Download the APM Server Windows zip file from the \
+[Download page]({downloadPageLink}).\n2. Extract the contents of \
+the zip file into {zipFileExtractFolder}.\n3. Rename the {apmServerDirectory} \
+directory to `APM-Server`.\n4. Open a PowerShell prompt as an Administrator \
+(right-click the PowerShell icon and select \
+**Run As Administrator**). If you are running Windows XP, you might need to download and install \
+PowerShell.\n5. From the PowerShell prompt, run the following commands to install APM Server as a Windows service:',
+        values: {
+          downloadPageLink: 'https://www.elastic.co/downloads/apm/apm-server',
+          zipFileExtractFolder: '`C:\\Program Files`',
+          apmServerDirectory: '`apm-server-{config.kibana.version}-windows`',
+        }
+      }),
       commands: [
         `PS > cd 'C:\\Program Files\\APM-Server'`,
         `PS C:\\Program Files\\APM-Server> .\\install-service-apm-server.ps1`,
       ],
-      textPost:
-        'Note: If script execution is disabled on your system, you need to set the execution policy for the current session' +
-        ' to allow the script to run. For example: `PowerShell.exe -ExecutionPolicy UnRestricted -File .\\install-service-apm-server.ps1`.',
+      textPost: i18n.translate('kbn.server.tutorials.apm.windowsServerInstructions.textPost', {
+        // eslint-disable-next-line no-multi-str
+        defaultMessage: 'Note: If script execution is disabled on your system, \
+you need to set the execution policy for the current session \
+to allow the script to run. For example: `PowerShell.exe -ExecutionPolicy UnRestricted -File .\\install-service-apm-server.ps1`.',
```

Comment:
Maybe we should move `PowerShell.exe -ExecutionPolicy UnRestricted -File .\\install-service-apm-server.ps1` to the `values`. What do you think?

Reply:
It's a good point. Thank you.
Fixed.